lxd/instance/drivers/driver_common.go

```diff
@@ -164,6 +164,10 @@ func (d *common) IsStateful() bool {
 	return d.stateful
 }
 
+func (d *common) Operation() *operations.Operation {
```

Comment:
Function should have a golint compliant comment (`Operation returns the operation associated with the instance`)

Reply:
@masnax while you're at it, can you also add a separate commit fixing the type in the function above?
`retuens` should be `returns` :)

Reply:
Interesting, I'm not getting any nags about the missing comment. Fixing now!

Reply:
Yeah, I'm not sure why it's not complaining, I would have expected it to...